gkeys/gkeys/actions.py

```diff
@@ -514,6 +547,31 @@ def speccheck(self, args):
                 'SPEC Approved..........: %d' % len(set(failed['spec-approved'])),
             ])
 
+    def _find_template_path(self):
+        config_dir = self.config.get_key('userconfigdir')
+        template_dir = os.path.join(config_dir + '/templates')
+        self.logger.debug("User template path is:" + template_dir)
+        template_path = '/usr/share/gkeys/templates/email_template'
+        if os.path.exists(template_dir + '/email_template'):
+            template_path = template_dir + '/email_template'
+        return template_path
```

Comment:
This whole function would not be required if you used the config system correctly.  Plus you are STILL HARD-CODING CONFIG VALUES IN CODE.